orchestration/templates/ingest-hca.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: ingest-hca
+spec:
+  entrypoint: main
+  serviceAccountName: {{ .Values.serviceAccount.k8sName }}
+  templates:
+    - name: main
+      inputs:
+        parameters:
+          - name: source-bucket-name
+          - name: staging-bucket-name
+
+      dag:
+        tasks:
+          # Pre-process the data found in a source bucket
+          - name: pre-process-source
+            templateRef:
+              name: run-dataflow
+              template: main
+            arguments:
+              parameters:
+                - name: source-bucket-name
+                  value: '{{ "{{inputs.parameters.source-bucket-name}}" }}'
+                - name: staging-bucket-name
+                  value: '{{ "{{inputs.parameters.staging-bucket-name}}" }}'
+
+    # Template used to launch a Dataflow processing job on raw ClinVar data,
+    # transforming it to our target schema.
+    - name: run-dataflow
+      inputs:
+        parameters:
+          - name: source-bucket-name
+          - name: staging-bucket-name
```

Comment:
If we decide to make this a Helm var, would remove this parameter and inline the Helm lookup where it's used below.